lib/pages/chat_details/chat_details.dart

```diff
@@ -147,6 +147,8 @@ class ChatDetailsController extends State<ChatDetails>
     });
     room = Matrix.of(context).client.getRoomById(roomId!);
     muteNotifier.value = room?.pushRuleState ?? PushRuleState.notify;
+    membersNotifier.value ??=
+        Matrix.of(context).client.getRoomById(roomId!)!.getParticipants();
```

Comment:
```suggestion
    membersNotifier.value ??= room.getParticipants();
```

- What do you think?
- separate a func for this?

Reply:
Not resolved. Line 157 room is already defined by `Matrix.of(context).client.getRoomById(roomId!)`

Reply:
done